test/utils/image/manifest.go

```diff
@@ -229,7 +234,7 @@ const (
 	WindowsServer
 )
 
-func initImageConfigs(list RegistryList) (map[ImageID]Config, map[ImageID]Config) {
+func initImageConfigs(list *RegistryList) (map[ImageID]Config, map[ImageID]Config) {
```

Comment:
This doesn't seem necessary?

Just keep copying `RegistryList` by value, it is small enough for that.